packages/cactus-plugin-ledger-connector-fabric/src/test/typescript/integration/fabric-v2-2-x/deploy-lock-asset.test.ts

```diff
@@ -1,13 +1,21 @@
-import { AddressInfo } from "net";
```

Comment:
Can we rename this file to convey information that it holds utility function method tests as well?

Reply:
> Can we rename this file to convey information that it holds utility function method tests as well?

@outSH Agreed. I'll move the utility functions out of it to their own separate files and also find some more generic name for the test case file because now it's not just about deploying the lock asset!

Reply:
@outSH Pushed the changes just now, let me know what you think!